document_definition/README.rst

```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+===================
+document_definition
+===================
+
+Installs a new  document_definition model for dictionary-style word archive and
+definition. Searchable in the Knowledge menu.
```

Comment:
Searchable on which view? Can you describe the steps a user would do to use the functionality of this module as per [here](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst#usage)